test/fuzz/oss_fuzz_build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -eu
```

Comment:
Do we need this script? I don't see anything calling it.

Reply:
It's called by oss-fuzz

Reply:
```suggestion
#!/bin/bash -eu
# DO NOT DELETE: this script is used from oss-fuzz.
# You can find more details in the official documentation: https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/
```

Reply:
Added a comment directly